app/apps/cases/views.py

```diff
@@ -50,9 +54,113 @@ class CaseViewSet(
     RetrieveUpdateDestroyAPIView,
     FinesMixin,
     DebriefingsMixin,
-    CaseEventsMixin,
 ):
     permission_classes = [IsAuthenticated]
     serializer_class = CaseSerializer
     queryset = Case.objects.all()
     filterset_class = CaseFilter
+
+    @action(detail=False, methods=["get"])
+    def mock_cases(self, request):
+        start_date = datetime.datetime.now().replace(minute=0, second=0, microsecond=0)
+
+        cases = baker.make(Case, start_date=datetime.date.today(), _quantity=5)
+
+        for case in cases:
+            baker.make(
+                CaseState, status__name="In looplijst", state_date=start_date, case=case
+            )
+
+        baker.make(
+            Visit,
+            case=cases[1],
+            start_time=start_date.replace(hour=10),
+            situation=Visit.SITUATION_NOBODY_PRESENT,
+            suggest_next_visit="weekend",
+            suggest_next_visit_description="Ziet er uit als feesthuis. Grote pakkans in het weekend",
+            observations=[
```

Comment:
je zou de observations in een mock_observations lijst kunnen zetten aangezien ze toch overal hetzelfde zijn. Is misschien iets cleaner :)

Reply:
Komt doordat het een dezelfde case is met iedere keer een extra stap. zal het iig cleaner maken